packager/media/trick_play/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Google LLC. All rights reserved.
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+add_library(media_trick_play STATIC
```

Comment:
Please delete the corresponding .gyp file as well.  Thanks!

Reply:
Yeah, I forgot 🙂